readme.md

```diff
@@ -232,16 +239,18 @@ rm -r my_useless_folder
 **STOP OPENING YOUR FINDER OR FILE EXPLORER** :-1:
 
 ```shell
-ls -la my_folder
+ls my_folder        # Simple
```

Comment:
I don't mind adding this. But I actually rarely use it. I'm wondering if we can add some other flags that might be useful?

Reply:
Sure! I most often use `ls -alrth` to show a list of files, sorted by time (reversed - most recently edited at the bottom), with human-readable sizing.